src/features/defi/components/DefiModalHeader/DefiModalHeader.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box } from '@chakra-ui/layout'
+import { Image } from '@chakra-ui/react'
+import { Text } from 'components/Text'
+
+type DefiModalHeaderProps = {
+  headerImageSrc: string
+  headerText: string | [string, Record<string, string>]
+}
+
+export const DefiModalHeader = ({ headerImageSrc, headerText }: DefiModalHeaderProps) => (
```

Comment:
As noted on Discord, IMO it doesn't make sense to over-engineer this and start creating atoms/molecules/organisms folders. We can have an atomic design-system pattern with a flat structure at first, and slowly start to collocate compoonents/move the more generic ones into the right folders.

Reply:
Start simple, then divide into atoms/molecules when it gets reusable is a great philosophy I think so